src/Services/RemoteWeb.php

```diff
@@ -427,10 +429,18 @@ protected function processRequest()
             $error = Curl::getError();
             $code = array_get($error, 'code', 500);
```

Comment:
Just wondering. if you do a dd() on the $error variable, does it contain anything else except the 'code'?  It it has a message maybe we can just use that.